docs/SUMMARY.md

```diff
@@ -18,6 +18,7 @@
 * [Rapid development with Kubernetes](tutorials/kubernetes-rapid.md)
 * [Get started with OpenShift](tutorials/openshift.md)
 * [Minikube VPN access](tutorials/minikube-vpn.md)
+* [Using Telepresence with Golang](tutorials/kubernetes-client-libs.md)
```

Comment:
This link could say something like "Using a Kubernetes Client Library"

Reply:
I forgot it, now fixed. Thank you!